### ISSUE TITLE
Initial game story / objective

### ISSUE THREAD

**Issue by frankhale** [ybotspawn/BeerTrek#8]
```
The game will provide a backstory to provide insight to the player on what to expect.
```
Somebody needs to come up with the initial backstory and overall objective for the game. This text will be displayed in the instructions part and will also help to drive the game development.

**Comment by frankhale** [comment]
I redacted some requirements that would deal with the scope of the game due to time constraints. A large scope in the story would contradict the changes. I was figuring that there would be one away mission and the player would have to navigate the would to get the items and the make it back to be beamed back. This would conclude the game.

**Comment by frankhale** [comment]
Navigate the world that is

**Comment by frankhale** [comment]
Thoughts?

**Comment by frankhale** [comment]
Closed via c541cd0c0733568aa4c8c3a06072e18a69edd8c9
